packages/core/src/core_render3_private_export.ts

```diff
@@ -48,6 +48,7 @@ export {
 export {
   AttributeMarker as ɵAttributeMarker,
   ComponentDef as ɵComponentDef,
+  ComponentDebugMetadata as ɵComponentDefDebugMetadata,
```

Comment:
Should we expose it under the same name, just with the `ɵ` in front of it?
```suggestion
  ComponentDebugMetadata as ɵComponentDebugMetadata,
```

Reply:
Good catch